{"name": "test_zero_volume_cases[4h-CES-DAI]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 429 == 200\n +  where 429 = <Response [429]>.status_code", "trace": "tests/special/test_edge_cases.py:16: in test_zero_volume_cases\n    assert response.status_code == 200\nE   assert 429 == 200\nE    +  where 429 = <Response [429]>.status_code"}, "parameters": [{"name": "timeframe", "value": "'4h'"}, {"name": "pair", "value": "'CES-DAI'"}], "start": 1758131943253, "stop": 1758131943505, "uuid": "0b5c38c6-2ba1-4edb-91b5-b6269a107101", "historyId": "83b39b7d82deebc809f873e183ec338b", "testCaseId": "f1061a3867aae255684da5f0a2754e42", "fullName": "tests.special.test_edge_cases#test_zero_volume_cases", "labels": [{"name": "tag", "value": "special"}, {"name": "parentSuite", "value": "tests.special"}, {"name": "suite", "value": "test_edge_cases"}, {"name": "host", "value": "826d662bff36"}, {"name": "thread", "value": "2128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.special.test_edge_cases"}], "titlePath": ["tests", "special", "test_edge_cases.py"]}